login.css

```diff
@@ -79,53 +53,65 @@ nav a{
 }
 
 
+ul li ul{
+	padding: 0;
+	position: absolute;
+	top: 58px;
+	left: 130px;
+	width: 100px;
+	display: none;
+	visibility: hidden;
+}
 
-.content_1{
+/* Set up position for content div to hold the login function */
+.content{
 	background-color: white;
+	z-index:4;
+	height:680px;
 	grid-column-start: 1;
-	grid-column-end: span 4;
-	z-index: 2;	
-	height: 100px;
-	border: 2px solid red;
+	grid-column-end: span 1;
+	
 }
 
-.section3{
-	background-color: white;
-	width: 400px;
-	height: 300px;
-	padding: 10px;
-	position: absolute;
-	top: 550px;
-	left: 35%;
-	z-index: 4;
-}
 
-.content{
+.img_2{
 	background-color: white;
 	z-index: 2;
 	height:680px;
+	grid-column-start: 2;
+	grid-column-end: span 4;
+}
+
+.img_2 img{
+	height:680px;
+	width:100%;
+}
+/* Set up the footer */
+.footer{
```

Comment:
01/08/2021: I setting the color and the position of the footer. For now it is just black and white(place hold)